pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -1945,8 +1945,13 @@ public boolean getBoolean(int columnIndex) throws SQLException {
       return false; // SQL NULL
     }
 
+    int col = columnIndex - 1;
+    if (Oid.BOOL == fields[col].getOID()) {
+      final byte[] value = this_row[col];
+      return (value.length == 1) && ('t' == (char) (value[0] & 0xFF));
```

Comment:
is this "byte to char" conversion necessary?
`(byte) 't'` looks shorter and overall protocol is defined in bytes, not chars.